blog/md/ref/jenkins/jenkins_day1-3_oluchi.md

```diff
@@ -0,0 +1,136 @@
+# Learning Jenkins: My 3-Day Adventure as a Beginner DevOps Engineer
+
+## Day 1 – Installing Jenkins and Creating My First Job
+
+The Jenkins WAR file was downloaded and executed locally.
+
+I started by setting up Jenkins using Docker:
+
+![Jenkins setup screenshot](https://miro.medium.com/v2/resize:fit:1100/format:webp/0*3vqP5FoKp1rr_wfS)
```

Comment:
can you take a screenshot and insert here using ctrl+v. In windows you can use snipet, and in mac it is cmd+shift+4.